components/Buttons/PlayButton.vue

```diff
@@ -43,32 +47,57 @@ export default Vue.extend({
       type: Array as () => BaseItemDto[],
       required: true
     },
+    disabled: {
+      type: Boolean,
+      required: false,
+      default: false
+    },
     fab: {
       type: Boolean
     },
     shuffle: {
       type: Boolean
     }
   },
+  data() {
+    return {
+      loading: false
+    };
+  },
+  computed: {
+    ...mapState('playbackManager', ['status'])
+  },
+  watch: {
+    status(): void {
+      if (this.status === PlaybackStatus.playing) {
+        this.loading = false;
+      }
+    }
+  },
   methods: {
     ...mapActions('playbackManager', ['play']),
     playOrResume(): void {
-      if (this.items.length > 0 && this.canResume(this.items[0])) {
-        this.play({
-          items: this.items,
-          startFromTime:
-            this.ticksToMs(this.items[0].UserData?.PlaybackPositionTicks) / 1000
-        });
-      } else if (this.shuffle) {
-        // We force playback from the start when shuffling, since you wouldn't resume AND shuffle at the same time
-        this.play({
-          items: this.items,
-          startShuffled: true
-        });
-      } else {
-        this.play({
-          items: this.items
-        });
+      if (!this.disabled) {
```

Comment:
This isn't necessary. If the buttons are disabled, the click event is never sent (Tested on master).